kubectl-cloudflow/cmd/deploy.go

```diff
@@ -132,6 +123,36 @@ You can update the credentials with the "update-docker-credentials" command.
 	rootCmd.AddCommand(deployOpts.cmd)
 }
 
+// validateVerifyCmdArgs validates the args for the verify command.
+// we only accept one argument at the moment which is a path to the blueprint.
+func validateDeployCmdArgs(cmd *cobra.Command, args []string) error {
+	if cmd.Flag("image").Changed == false && cmd.Flag("blueprint").Changed == false {
+		return fmt.Errorf("Please specify either the blueprint file or the image path")
```

Comment:
Does it print the example in the help above when the validation fails? I think that would be helpful.

Reply:
@olofwalker yes, it does ..

```
$ ./kubectl-cloudflow deploy --image img --blueprint record-aggregator-blueprint.conf -u oauth2accesstoken -p "$(gcloud auth print-access-token)" 
Error: Cannot specify both blueprint and image in command
Usage:
  cloudflow deploy [flags]

Examples:

> kubectl cloudflow deploy --image registry.test-cluster.io/cloudflow/sensor-data-scala:292-c183d80 valid-logger.log-level=info valid-logger.msg-prefix=valid
> kubectl cloudflow deploy --blueprint call-record-aggregator-blueprint.conf

Flags:
  -b, --blueprint string           blueprint file path.
  -h, --help                       help for deploy
  -i, --image string               docker image path.
  -p, --password string            docker registry password.
      --password-stdin             Take the password from stdin
      --scale stringToInt          Accepts key/value pairs for replicas per streamlet (default [])
  -u, --username string            docker registry username.
      --volume-mount stringArray   Accepts a key/value pair separated by an equal sign. The key should be the name of the volume mount, specified as '[streamlet-name].[volume-mount-name]'. The value should be the name of an existing persistent volume claim.

Cannot specify both blueprint and image in command
```